test/contracts/bridge.ts

```diff
@@ -1,66 +1,88 @@
-import { OptimismAddresses, waitForTx } from './helpers'
-import { getAddressOfNextDeployedContract } from './pe-utils/address'
+} from '../../typechain'
+import { L2AddWormholeDomainSpell__factory } from '../../typechain/factories/L2AddWormholeDomainSpell__factory'
+import { deployUsingFactory, getContractFactory, mintEther } from '../helpers'
+import { OptimismAddresses, waitForTx } from '../helpers'
+import { getAddressOfNextDeployedContract } from '../pe-utils/address'
+import { WormholeSdk } from './wormholeJoin'
 
 interface BridgeDeployOpts {
   l1Signer: Signer
   l2Signer: Signer
   mainnetSdk: MainnetSdk
+  wormholeSdk: WormholeSdk
+  baseBridgeSdk: BaseBridgeSdk
   optimismAddresses: OptimismAddresses
   domain: string
-  wormholeRouter: string
-  l1Escrow: Wallet
-  l2Dai: Dai
 }
 
 export async function deployBridge(opts: BridgeDeployOpts) {
   const futureL1WormholeBridgeAddress = await getAddressOfNextDeployedContract(opts.l1Signer)
   const L2WormholeBridgeFactory = getContractFactory<L2DAIWormholeBridge__factory>('L2DAIWormholeBridge', opts.l2Signer)
   const l2WormholeBridge = await deployUsingFactory(opts.l2Signer, L2WormholeBridgeFactory, [
     opts.optimismAddresses.l2.xDomainMessenger,
-    opts.l2Dai.address,
+    opts.baseBridgeSdk.l2Dai.address,
     futureL1WormholeBridgeAddress,
     opts.domain,
   ])
-  // wormhole bridge has to have burn rights
-  await opts.l2Dai.rely(l2WormholeBridge.address)
 
   const L1WormholeBridgeFactory = getContractFactory<L1DAIWormholeBridge__factory>('L1DAIWormholeBridge')
   const l1WormholeBridge = await deployUsingFactory(opts.l1Signer, L1WormholeBridgeFactory, [
     opts.mainnetSdk.dai.address,
     l2WormholeBridge.address,
     opts.optimismAddresses.l1.xDomainMessenger,
-    opts.l1Escrow.address,
-    opts.wormholeRouter,
+    opts.baseBridgeSdk.l1Escrow.address,
+    opts.wormholeSdk.router.address,
   ])
   expect(l1WormholeBridge.address).to.be.eq(futureL1WormholeBridgeAddress, 'Future address doesnt match actual address')
-  await opts.mainnetSdk.dai.connect(opts.l1Escrow).approve(l1WormholeBridge.address, constants.MaxUint256)
 
   return { l2WormholeBridge, l1WormholeBridge }
 }
+export type BridgeSdk = Awaited<ReturnType<typeof deployBridge>>
+
+export async function configureWormholeBridge({
+  l2Signer,
+  baseBridgeSdk,
+  bridgeSdk,
+  masterDomain,
+}: {
+  l2Signer: Signer
+  bridgeSdk: BridgeSdk
+  baseBridgeSdk: BaseBridgeSdk
+  masterDomain: string
+}) {
+  const l2AddWormholeSpell = await deployUsingFactory(
+    l2Signer,
+    getContractFactory<L2AddWormholeDomainSpell__factory>('L2AddWormholeDomainSpell'),
+    [baseBridgeSdk.l2Dai.address, bridgeSdk.l2WormholeBridge.address, masterDomain],
+  )
+
+  // we can do this b/c we didn't configure full fledged governance on L2
+  await bridgeSdk.l2WormholeBridge.rely(l2AddWormholeSpell.address)
+  await baseBridgeSdk.l2Dai.rely(l2AddWormholeSpell.address)
+  await waitForTx(l2AddWormholeSpell.execute())
```

Comment:
TODO: deny deployer, rely governanceRelay on l2WormholeBridge?

Reply:
I'll do it in the next PR as we don't deploy now gov relay....